### PR TITLE
chore: add Gatsby unified domain to scraping

### PR DESCRIPTION
**Pull request by DSchau**
# Pull request motivation(s)

We're hoping to additionally scrape our Cloud documentation (gatsbyjs.com) by August 12th, which will live at gatsbyjs.com/docs/cloud and we'll have additional docs on this page, many of which are currently on the open-source documentation page (so please, please tag me in if I can help with any kind of approval / de-duping of these duplicate results!).

We'll be deploying the updates to the site by end-of-week (August 7th), which will give you all a few days to ensure we have data flowing through. Please feel free to tag me in if I can help with anything, or if we need to remove any duplicates or that kind of thing before the launch date.

cc @s-pace as info

## Additional Context

Opened a support ticket with Algolia (Matthew Bond) who shared the following info:

> To make sure there is no interruption on the search experience, please submit a PR against your DocSearch configuration with the new domain. Please mention in the PR's body the exact date when the documentation will be available from the .com website.

> The most efficient way to handle this move would be to do a silent release on your .com website. It will enable us to crawl the website first. We can merge the PR within a day if the PR is submitted during the week. Please tag the Github handle @s-pace in the PR.

### CODE REVIEW

File: configs/gatsbyjs.json [hunk 1]
```diff
@@ -20,10 +20,37 @@
...
+    {
+      "url": "https://www.gatsbyjs.com/docs/",
+      "tags": [
+        "docs"
+      ]
+    },
+    {
+      "url": "https://www.gatsbyjs.com/cloud/docs/",
+      "tags": [
+        "docs"
```
Comment (DSchau) [inline configs/gatsbyjs.json:33]: Unsure if this should be a separate tag? (actually not sure _how_ this works). Feedback welcome!

Comment (s-pace) [reply configs/gatsbyjs.json:33]: Should be good this way

**Comment by s-pace** [comment]
👋 @DSchau 

Are you ready? Can we merge this?

**Comment by DSchau** [comment]
@s-pace ready our end, at least from a content perspective.

Question -- what happens when this is merged? gatsbyjs.com/docs/ is largely duplicative of gatsbyjs.org/docs/? Is there some type of de-duping automatically?

**Comment by s-pace** [comment]
There is no de-duping system. Is there any content from gatsbyjs.com/docs not available in gatsbyjs.org/docs or the other way around?

**Comment by DSchau** [comment]
It's the same content! Should we remove the gatsbyjs.org/docs/ from this PR, then?

**Comment by s-pace** [comment]
I have updated the config and the run a crawl accordingly. You should be ready to go :)

**Comment by DSchau** [comment]
@s-pace thank you 🙏 I really appreciate your help on this. A+ from Algolia 💜

**Comment by DSchau** [comment]
@s-pace one more question -- do the links change? e.g. from instantsearch on .org will the links go to .com? (If so -- we'll definitely want to ensure that is not the case)

**Comment by s-pace** [comment]
Thank you for your feedback.

The URLs do change. The scraper uses the URL of the crawled page. You might want to leverage the[ transformData method on the .org website](https://docsearch.algolia.com/docs/behavior/#transformdata). It will update the results on the client side. I have [built a showcase environment to let you play with it.](https://jsfiddle.net/s_pace/1kLjqo3w/)

I have also tailored your [configuration to enhance the contextual information displayed for each result. It improves the results extracted from the Gatsby cloud documentation](https://github.com/algolia/docsearch-configs/commit/ca30307ed7941f722c94533e1c16fc84eb431369). 

Let us know if you need anything else

**Pull request closed without merging**